components/calendar/src/types.rs

```diff
@@ -97,7 +97,7 @@ impl MonthCode {
     /// Returns an option which is Some containing the non-month version of a leap month
     /// if the MonthCode this method is called upon is a leap month, and None otherwise.
     /// This method assumes the MonthCode is valid.
-    pub fn get_normal_if_leap(self) -> Option<MonthCode> {
+    pub fn get_normal_if_leap(&self) -> Option<MonthCode> {
```

Comment:
nit: MonthCode is Copy, this should just take `self`

Reply:
Done